streams-contrib/streams-provider-twitter/src/test/java/org/apache/streams/twitter/processor/FetchAndReplaceTwitterProcessorTest.java

```diff
@@ -0,0 +1,48 @@
+    private Activity activity = null;
+
+    @Test
+    public void testLocationReplacement() throws IOException, ActivitySerializerException {
+        Map<String, Object> extensions = Maps.newHashMap();
+        Map<String, Object> location = Maps.newHashMap();
+
+        extensions.put("location", location);
+        location.put("country", "us");
+
+        FetchAndReplaceTwitterProcessor fetchAndReplaceTwitterProcessor = new FetchAndReplaceTwitterProcessor();
+        fetchAndReplaceTwitterProcessor.prepare(null);
+
+        activity = new Activity();
+        activity.setAdditionalProperty("extensions", extensions);
+
+        fetchAndReplaceTwitterProcessor.replace(activity, json);
+
+        String country = getCountry(activity);
+        Assert.assertTrue(country.equals("us"));
```

Comment:
Please use assertEquals() or assertThat() to test equality.  That way if it fails, the message will spit out the values that weren't equal.